### DISCUSSION TITLE
How do I update the vod table

### DISCUSSION THREAD

**Discussion by DGNVMusic** [saplinganon/imissfauna.com#6] [Q&A] [answered]
I'm trying to use the project but for Ayame instead, but I can't seem to get the postgres tables updated with the actual content. It's currently still stuck on the dummy info. Although the stream tracker works as intended with the right data being pulled from the YouTube API, I can't get the `/reps` path to work. (Message I get is that there is no video found) which could be because the vod table in the database is empty and not updated. Do I need to run a certain js file standalone? Or am I being dumb and overlooking something?

Thanks for the help in advance. Love the project.

**Comment by saplinganon** [comment, marked as answer]
You need to run `tool/updateVODList.js` with node to update the VOD table.

(I'd probably make a separate YouTube API key for this because it fetches every video on the channel, so you might end up going over quota for large channels and locking yourself out for the day.)

**Reply by DGNVMusic** [reply]
I'll try it out! Thank you. 

Also happy new year
